webpack/assets/javascripts/react_app/components/SearchBar/SearchBarHelpers.js

```diff
@@ -1,8 +1,7 @@
-import URI from 'urijs';
+import { getURI, changeQuery } from '../Pagination/PaginationHelper';
```

Comment:
* Isn't it odd you are importing `getURI` and `changeQuery` from `PaginationHelper`?
* isn't it suppose to use `react-router` now for those type of operations?

Reply:
this function is for the current views not for the `react-router` ones. either way I think its best to move all URI functions to a dedicated helper.